Dev dependencies do not support Node 12

```
dmurvihill@penguin:~/code/postgres-date$ node --version
v12.22.12
dmurvihill@penguin:~/code/postgres-date$ npm i
npm WARN deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm WARN deprecated eslint@8.57.1: This version is no longer supported. Please see https://eslint.org/version-support for other options.
npm WARN deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm WARN deprecated @humanwhocodes/config-array@0.13.0: Use @eslint/config-array instead
npm WARN deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead
npm WARN deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN notsup Unsupported engine for tsd@0.28.1: wanted: {"node":">=14.16"} (current: {"node":"12.22.12","npm":"6.14.16"})
npm WARN notsup Not compatible with your version of node/npm: tsd@0.28.1
npm WARN notsup Unsupported engine for jest-diff@29.7.0: wanted: {"node":"^14.15.0 || ^16.10.0 || >=18.0.0"} (current: {"node":"12.22.12","npm":"6.14.16"})
npm WARN notsup Not compatible with your version of node/npm: jest-diff@29.7.0
npm WARN notsup Unsupported engine for pretty-format@29.7.0: wanted: {"node":"^14.15.0 || ^16.10.0 || >=18.0.0"} (current: {"node":"12.22.12","npm":"6.14.16"})
npm WARN notsup Not compatible with your version of node/npm: pretty-format@29.7.0
npm WARN notsup Unsupported engine for jest-get-type@29.6.3: wanted: {"node":"^14.15.0 || ^16.10.0 || >=18.0.0"} (current: {"node":"12.22.12","npm":"6.14.16"})
npm WARN notsup Not compatible with your version of node/npm: jest-get-type@29.6.3
npm WARN notsup Unsupported engine for diff-sequences@29.6.3: wanted: {"node":"^14.15.0 || ^16.10.0 || >=18.0.0"} (current: {"node":"12.22.12","npm":"6.14.16"})
npm WARN notsup Not compatible with your version of node/npm: diff-sequences@29.6.3
npm WARN notsup Unsupported engine for @jest/schemas@29.6.3: wanted: {"node":"^14.15.0 || ^16.10.0 || >=18.0.0"} (current: {"node":"12.22.12","npm":"6.14.16"})
npm WARN notsup Not compatible with your version of node/npm: @jest/schemas@29.6.3

added 365 packages from 153 contributors and audited 365 packages in 20.471s

143 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

`pretty-format` at least is not working on Node 12:
```
dmurvihill@penguin:~/code/postgres-date$ npm test

> postgres-date@2.1.0 test /home/dmurvihill/code/postgres-date
> standard && tape test.js && tsd

TAP version 13
# date parser
ok 1 should be strictly equal
ok 2 should be strictly equal
ok 3 should be strictly equal
ok 4 should be strictly equal
ok 5 should be strictly equal
ok 6 should be strictly equal
ok 7 should be strictly equal
ok 8 should be strictly equal
ok 9 should be strictly equal
ok 10 should be strictly equal
ok 11 no timezones
ok 12 huge ms value
ok 13 zulu time offset
ok 14 negative hour offset
ok 15 positive HH:mm offset
ok 16 negative HH:mm offset
ok 17 positive HH:mm:ss offset
ok 18 negative HH:mm:ss offset
ok 19 0 to 99 year boundary
ok 20 should be strictly equal
ok 21 should be strictly equal

1..21
# tests 21
# pass  21

# ok

/home/dmurvihill/code/postgres-date/node_modules/pretty-format/build/index.js:412
  options?.printFunctionName ?? DEFAULT_OPTIONS.printFunctionName;
          ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/dmurvihill/code/postgres-date/node_modules/jest-diff/build/index.js:63:21)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
npm ERR! Test failed.  See above for more details.
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -6,12 +6,12 @@ jobs:
     strategy:
       matrix:
         node-version:
-        - '12'
-        - '14'
+        - '20'
+        - 'lts/*'
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v4
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "url": "bendrucker.me"
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=20"
   },
   "scripts": {
     "test": "standard && tape test.js && tsd"

